src/Client.php

```diff
@@ -214,17 +214,12 @@ public function createClientEncryption(array $options): ClientEncryption
      * @see DropDatabase::__construct() for supported options
      * @param string $databaseName Database name
      * @param array  $options      Additional options
-     * @return array|object Command result document
      * @throws UnsupportedException if options are unsupported on the selected server
      * @throws InvalidArgumentException for parameter/option parsing errors
      * @throws DriverRuntimeException for other driver errors (e.g. connection errors)
      */
-    public function dropDatabase(string $databaseName, array $options = []): array|object
+    public function dropDatabase(string $databaseName, array $options = []): void
```

Comment:
Should these changes come with a `ReturnTypeWillChange` attribute in 1.x or is that attribute only designed to suppress notifications from PHP core?

Reply:
Shame, but good to know.